tutorial/gue/out/gue0.out

```diff
@@ -0,0 +1,4 @@
+GoodJobYouDiDEndTheGame!!
```

Comment:
Dlaczego w przykładowym outcie jest sekret z biblioteczki?

Reply:
Przykładowe outy nie są widoczne dla uczestników, jeśli autor zadania sobie tego nie zażyczy. Stąd nie mają oni możliwości znaleźć tego sekretu. Zaś jest to potrzebne, by checkerka poprawnie zadziałała dla sampla